src/styles/blocks/card.scss

```diff
@@ -0,0 +1,98 @@
+@use "../utils/variable" as *;
+
+.card {
+  width: 200px;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-color: $bg-color-white;
+  border: 1px solid $border-card;
+  border-radius: 5px;
+
+  &:not(:nth-last-child(-n + 4)) {
+    margin-bottom: 46px;
+  }
+
+  &:not(:nth-child(4n)) {
+    margin-right: 48px;
+  }
+
+  &__picture {
+    width: 160px;
+    height: 134px;
+    margin-top: 16px;
+  }
+
+  &__title {
+    margin-top: 40px;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $text-main-color;
+  }
+
+  &__code {
+    margin-top: 4px;
+    font-size: 10px;
+    line-height: 14px;
+    color: $text-title-color;
+    align-self: flex-start;
+  }
+
+  &__stars {
+    margin-top: 16px;
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    margin-top: 24px;
+    display: flex;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+  &__text {
+    font-size: 12px;
+    line-height: 18px;
+    color: $text-title-color;
+  }
+
+  &__cost {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $text-main-color;
+  }
+
+  &__button {
+    margin-top: 16px;
+    padding: 11px 0;
+    width: 100%;
+    background: $text-main-blue;
+    border: 1px solid $text-main-blue;
+    border-radius: 5px;
+    color: $bg-color-white;
+    font-weight: 700;
+    font-size: 14px;
+    line-height: 16px;
+    text-align: center;
+    text-transform: uppercase;
+    text-decoration: none;
+  }
+
+  &__button:hover {
+    background: $bg-color-white;
+    color: $text-main-blue;
+  }
+
+  &__review {
+    align-self: center;
```

Comment:
add to `card__stars` style `align-items: center;` and you can remove this `align-self`